core-sdk-samples/mp-sideloaded-kit-example/dist/SampleSideLoadedKit-Kit.common.js

```diff
@@ -0,0 +1,683 @@
+'use strict';
```

Comment:
Should we have a build task that produces these, rather than have them get committed?